Add docs for custom floating ip network

phiphi282 opened the pull request:
Describe how to change the nginx ingress configuration to use a custom
floating ip network.

Explain that this is not needed when using the default metakube network.

phiphi282 committed:
Add docs for custom floating ip network

Describe how to change the nginx ingress configuration to use a custom
floating ip network.
Explain that this is not needed when using the default metakube network.

stefanandres commented on pages/04.tutorials/27.setup-an-nginx-ingress-controller-with-the-proxy-protocol/default.en.md:
```diff
@@ -112,6 +112,23 @@ The example below configures the nginx ingress controller after the first update
+    enabled: true
+
+    annotations:
+      loadbalancer.openstack.org/floating-network-id: "*****"
```
Is this something only for octavia or can this be put into the general doc like https://docs.syseleven.de/metakube/en/tutorials/create-a-load-balancer#keeping-load-balancer-floating-ips-on-openstack?

phiphi282 replied:
Ah good catch. This is a general annotation that can be used in general. 👍

stefanandres replied:
then it should suffice in the general doc and this can be removed here :)

phiphi282 replied:
Linked to the generic page now instead :)

phiphi282 committed:
Add custom floating ip option to service tutorial

Add docs on how to set a custom floating ip network on a loadbalancer
service in general.

stefanandres commented on pages/04.tutorials/13.create-a-load-balancer/default.en.md:
```diff
@@ -40,3 +40,22 @@ loadbalancer   LoadBalancer   10.10.10.202   195.192.128.46   80:31228/TCP,443:3
 This load balancer now exposes port 80 to the outside world and maps it to port 31228 on all cluster nodes, and it exposes port 443 to the outside world and maps it to ports 30279 on all cluster nodes. This means that a loadbalancer service is also a NodePort service (i.e. a service that exposes pods on specific "NodePorts" on all nodes).
+
+
+## Additional configuration options. Use a custom floating ip network (optional)
```
I would keep it simple:

```suggestion
## Use a custom floating ip network
```

phiphi282 committed:
Remove fip docs from nginx-ingress

Link to the generic loadbalancer tutorial instead

stefanandres commented:
LGTM
